src/main/java/jdbchomework/utils/TestUtils.java

```diff
@@ -0,0 +1,20 @@
+package jdbchomework.utils;
+
+import jdbchomework.entity.Company;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class TestUtils {
+
+    public static final List<Company> ALL_COMPANIES = Arrays.asList(new Company("Test1"),
+            new Company("Test2"), new Company("Test3"));
+    public static final int ID = 111;
+    public static final String COMPANY_NAME = "Comp&Co";
+    public static final Company COMPANY_RESULT = new Company(COMPANY_NAME);
+
+
+//    private jdbchomework.utils.TestUtils() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)